### PR TITLE
Create red.json

### PR DESCRIPTION
**Pull request by StandRed**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x ] You're not using Vercel or Netlify.
- [x ] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: standred.dev
- [x ] The website is reachable.
- [ x] The CNAME record doesn't contain any slash.
- [x ] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/red.json
Comment (WilliamDavidHarrison) [inline domains/red.json:9]: Invalid record type.

File: domains/standred.json
Comment (WilliamDavidHarrison) [inline domains/standred.json:9]: Please provide a website link.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/red.json
Comment (WilliamDavidHarrison) [inline domains/red.json:9]: Please change this to `"URL": "https://www.strandred.dev"` if you want it to redirect.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/red.json
Comment (WilliamDavidHarrison) [inline domains/red.json:9]: Please add `https://` before www

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — DISMISSED** [review]

**Review by MaskDuck — APPROVED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
sorry, didn't mean to approve

**Comment by StandRed** [comment]
@MaskDuck @WilliamDavidHarrison i changed it for email forwarding

**Review by wdhdev — DISMISSED** [review]

**Comment by StandRed** [comment]
@WilliamDavidHarrison does it work now? im confused

**Comment by WilliamDavidHarrison** [comment]
> @WilliamDavidHarrison does it work now? im confused

Since it is now only email forwarding it's fine.

**Comment by StandRed** [comment]
> > @WilliamDavidHarrison does it work now? im confused
> 
> Since it is now only email forwarding it's fine.

so does the email work now?

**Comment by WilliamDavidHarrison** [comment]
> > > @WilliamDavidHarrison does it work now? im confused
> > 
> > 
> > Since it is now only email forwarding it's fine.
> 
> so is the email supposed to work now?

The email will work, but since you don't have a A record, there will be no website. So the domain will purely be for emails.

**Comment by StandRed** [comment]
i updated it @WilliamDavidHarrison may you merge please?

**Comment by WilliamDavidHarrison** [comment]
Sorry, I can't merge PR's, since I'm only a helper, you'll need to wait for a maintainer to merge it.

**Comment by StandRed** [comment]
> Sorry, I can't merge PR's, since I'm only a helper, you'll need to wait for a maintainer to merge it.

oh ok who are maintainers?

**Comment by StandRed** [comment]
also it closed randomly

**Comment by WilliamDavidHarrison** [comment]
> > Sorry, I can't merge PR's, since I'm only a helper, you'll need to wait for a maintainer to merge it.
> 
> oh ok who are maintainers?

Just be patient and they will eventually merge your pr.

**Comment by WilliamDavidHarrison** [comment]
> also it closed randomly

Just reopen it.

**Pull request closed without merging**
